content/docs/studio/user-guide/openid-connect.md

```diff
@@ -0,0 +1,283 @@
+# OpenID Connect (OIDC)
+
+DVC Studio can use OpenID Connect to access cloud resources securely, without
+requiring manual configuration of static credentials.
+
+## Cloud configuration
+
+- OpenID Connect Discovery URL:
+  https://studio.iterative.ai/api/.well-known/openid-configuration
```

Comment:
Does this or the terraform examples below need to change for self-hosted?

Reply:
@dberenbaum, this feature isn't[^1] compatible with Studio self-hosted.

[^1]: It could be, if the self-hosted deployment is publicly accessible from the internet, and if we document how to generate key pairs, but it's probably out of scope for now.